cookbook/how_to_pass_request_parameter_to_grid.rst

```diff
@@ -31,15 +31,116 @@ For example:
             # ...
         # ...
 
-Our goal is to set :contactId parameter with the value from a request.
+Our goal is to set :contactId parameter with the value from the request.
 
-Solution 1. Create custom event listener
-----------------------------------------
+Solution 1: Grid Parameter binding
```

Comment:
The new headlines are not very compatible with the old ones since the application logic has changed